### PR TITLE
New structure and design of the main interface

### PR DESCRIPTION
**Pull request by Gavalencia12**
#46  Update design and add responsiveness for devices
## Description
These changes include updates to the website's design and colors to enhance the user experience on mobile devices and other small-screen devices. Changes have been made to the CSS to ensure the design is responsive and adapts well to different screen sizes. Additionally, JavaScript functions have been added to improve usability on touch devices.
## List of Changes
- Updated colors in the interface palette.
- Adjusted section layouts for small-screen devices.
- Implemented CSS media queries to enhance adaptability.
- Added JavaScript functions to improve navigation on touch devices.
## Additional Information
image
image
image
image

### CODE REVIEW

File: resources/js/main.js
Comment (IsmaelMontiel710) [inline resources/js/main.js:1]: Delete the comments in spanish, please, I don´t understand spanish

File: resources/stylesheets/base/_base.scss
Comment (IsmaelMontiel710) [inline resources/stylesheets/base/_base.scss:1]: Also here

**Review by Zama15 — APPROVED** [review]
LGTM

File: resources/js/main.js
Comment (IsmaelMontiel710) [inline resources/js/main.js:1]: Delete the comments in spanish, please, I don´t understand spanish

File: resources/stylesheets/layout/_invi.scss
Comment (IsmaelMontiel710) [inline resources/stylesheets/layout/_invi.scss:1]: Character of more,  delete please

**Review by AngelDiaz591 — APPROVED** [review]

File: views/posts/index.php
Comment (IsmaelMontiel710) [inline views/posts/index.php:1]: Delete comments inapropriate

**Review by IsmaelMontiel710 — CHANGES_REQUESTED** [review]

**Pull request merged**